### DISCUSSION TITLE
Make simulate available from Chia CLI

### DISCUSSION THREAD

**Discussion by Fahrenheit2539** [Chia-Network/chia-blockchain#16270] [Ideas]
Right now bladebit simulate is available only from a standalone executable, which makes it hard to use on certain systems (for example there is no trivial way to run it from docker container).

Please make it available from the regular Chia CLI.

**Comment by BrandtH22** [comment]
Hey @Fahrenheit2539 , thank you for the recommendation. 
I have added a number of labels to make it easier to identify for our engineers as they plan out future releases 👍
